src/screens/DiscoveryScreenV2/elements/Search.js

```diff
@@ -226,48 +245,45 @@ const styles = StyleSheet.create({
   clearIconContainer: {
     justifyContent: 'center',
     alignItems: 'center',
-    marginRight: -20.5,
-    paddingRight: 30,
-    paddingLeft: 30,
+    marginRight: dimen.normalizeDimen(-20.5),
+    paddingHorizontal: dimen.normalizeDimen(30),
     zIndex: 1000
   },
   searchContainer: {
     flex: 1,
-    marginRight: 20
+    marginRight: dimen.normalizeDimen(20)
   },
   wrapperSearch: {
     flex: 1,
     backgroundColor: '#F5F5F5',
-    borderRadius: 8,
+    borderRadius: dimen.normalizeDimen(8),
     alignSelf: 'center',
     flexDirection: 'row',
-    height: 36,
-    paddingRight: 8
+    height: dimen.normalizeDimen(36),
+    paddingRight: dimen.normalizeDimen(8)
   },
   wrapperButton: {
     flexDirection: 'row',
     alignItems: 'center',
     justifyContent: 'space-around',
     marginEnd: SIZES.base,
-    paddingLeft: 8,
-    paddingRight: 8,
-    paddingTop: 9,
-    paddingBottom: 9
+    paddingHorizontal: dimen.normalizeDimen(8),
+    paddingVertical: dimen.normalizeDimen(9)
   },
   input: {
-    marginRight: 16,
-    paddingStart: 8,
+    marginRight: dimen.normalizeDimen(16),
+    paddingStart: dimen.normalizeDimen(8),
     flex: 1,
-    fontSize: 14,
+    fontSize: normalizeFontSize(14),
     fontFamily: fonts.inter[400],
     alignSelf: 'center',
-    height: 33,
+    height: dimen.normalizeDimen(33),
     paddingTop: 0,
     paddingBottom: 0
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Using the `dimen.normalizeDimen` function to set style values is a good practice for maintaining consistent layout and sizing across different screen sizes.
```diff
- marginRight: -20.5,
- paddingRight: 30,
- paddingLeft: 30,
+ marginRight: dimen.normalizeDimen(-20.5),
+ paddingHorizontal: dimen.normalizeDimen(30),

- marginRight: 20
+ marginRight: dimen.normalizeDimen(20)

- borderRadius: 8,
- height: 36,
- paddingRight: 8
+ borderRadius: dimen.normalizeDimen(8),
+ height: dimen.normalizeDimen(36),
+ paddingRight: dimen.normalizeDimen(8)

- paddingLeft: 8,
- paddingRight: 8,
- paddingTop: 9,
- paddingBottom: 9
+ paddingHorizontal: dimen.normalizeDimen(8),
+ paddingVertical: dimen.normalizeDimen(9)

- marginRight: 16,
- paddingStart: 8,
- fontSize: 14,
- height: 33,
+ marginRight: dimen.normalizeDimen(16),
+ paddingStart: dimen.normalizeDimen(8),
+ fontSize: normalizeFontSize(14),
+ height: dimen.normalizeDimen(33),

- marginLeft: 9.67,
- marginRight: 1.67,
+ marginLeft: dimen.normalizeDimen(9.67),
+ marginRight: dimen.normalizeDimen(1.67),
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->